### PR TITLE
Add explanation to the location of the user stories

### PR DESCRIPTION
**Pull request by karantan**

### CODE REVIEW

File: 2_Operations/user-stories.md [hunk 1]
```diff
@@ -2,9 +2,17 @@
 
 User Stories are well-defined tasks to be done by a Nitean that have a specific, measurable goal and impact. The vast majority of issues are written as User Stories, and only issues created by support that are reactionary and straightforward are not. Support issues are also not included in sprints.
 
+All user stories are located in the project repository:
```
Comment (dmurko) [inline 2_Operations/user-stories.md]: > All User Stories are located in their project repositories:

File: 2_Operations/user-stories.md [hunk 2]
```diff
@@ -2,9 +2,17 @@
...
+- EBN project: `niteoweb/easyblognetworks`
+- KAI project: `niteoweb/kai`
+- WooCart project: `niteoweb/woocart`
+- "Niteo project" (company stuff): `niteoweb/operations`
```
Comment (dmurko) [inline 2_Operations/user-stories.md]: > - Company issues: `niteoweb/operations`

File: 2_Operations/user-stories.md [hunk 3]
```diff
@@ -2,9 +2,17 @@
...
+
+All projects have multiple repositories (e.g. EBN https://github.com/niteoweb/easyblognetworks/#repositories) but only one project repository. The idea behind this is to have all user stories for one project in one repository.
```
Comment (dmurko) [inline 2_Operations/user-stories.md]: Re-read this, it doesn't make much sense.

Comment (karantan) [reply 2_Operations/user-stories.md]: We have many repositories for one project (e.g. ebn) but only one repository is the repository where we keep project user stories.

**Comment by zupo** [comment]
@dmurko, @karantan: can you polish up and merge this please?

File: 2_Operations/user-stories.md (hunk 3 above)
Comment (dmurko) [reply 2_Operations/user-stories.md]: > We have many repositories for each project but we keep User Stories in only one, main, repository.

**Pull request merged**